### ISSUE TITLE
Show current adjustment range in osd

### ISSUE THREAD

**Issue by jaimevisser** [basdelfos/beta-osd#1]
With inflight adjustments tuning is even easier then landing, disarming, using the osd to adjust settings.

Can we have either:
- A display of the currently active inflight adjustment index/slot with current value.
- Inflight adjustment functionality in the osd itself (then it can be stripped from betaflight in the future)

**Comment by ShikOfTheRa** [comment]
I know nothing about that functionality
Can you provide me with a link to descriptions of how it works?

For second... remember......
- menu access is not permitted when it is in active flight - to avoid accidentally turning it on. 
- menu access also requires stick manipulation - this might not be too good either
Not knowing about this functionality - it may be possible if can detect some mode?

There is code for changing profiles / pid controllers etc. but its not released yet as I've not tested it!

**Comment by basdelfos** [comment]
If i understand what you mean; what you want in essence is a visual conformation of PID/rates changes done by an external source (other that the OSD menu, e.g. with switch inputs from transmitter to flight controller). That way you don't have to count and remember every adjustment you make with switches.

That should be possible to make and not really hard to implement. The solution i have in mind is to display the changed parameter (PID or rates) with the new value and display this for about 2-3 seconds somewhere on the OSD screen, without entering the OSD menu.

Inflight adjustment functionality in the OSD itself is not possible, the OSD menu uses stick input to navigate through the menu and to increase/decrease values. Displaying the OSD menu inflight would crash you quad while navigating the menu.

**Comment by jaimevisser** [comment]
Yeah, the display would be a great addition already! Currently I have to figure out what adjustment I'm doing by taranis number callout and "feeling" if a value is ok or too twitchy.

Proposed way of showing it would be perfect!
